pkgs/tools/text/ugrep/default.nix

```
@@ -0,0 +1,26 @@
+{ fetchFromGitHub, stdenv, pcre2, boost, zlib, lzham }:
+
+let
+  pname = "ugrep";
+  version = "2.4.1";
+in stdenv.mkDerivation {
+  pname = pname;
+  version = version;
```

```suggestion
stdenv.mkDerivation rec {
  pname = "ugrep";
  version = "2.4.1";
```
you can do this. it's shorter and easier to read i think

---

I suggested this way following the advice from https://nix.dev/anti-patterns/index.html
However, it can in fact be made a bit nicer by using
```suggestion
let
  pname = "ugrep";
  version = "2.4.1";
in stdenv.mkDerivation {
  inherit pname version;
```

---

yes, that looks less redundant